core/metrics/src/test/scala/org/typelevel/otel4s/metrics/BaseMeterSuite.scala

```diff
@@ -138,7 +138,8 @@ abstract class BaseMeterSuite extends CatsEffectSuite {
     val duration = 100.nanos
     val expected = MetricData.histogram(
       "histogram",
-      List(duration.toNanos.toDouble)
+      List(duration.toNanos.toDouble),
+      exemplarValue = Some(100.00)
     )
```

Comment:
We can experiment with platform-specific condition using `import munit.internal.PlatformCompat`:
```scala
exemplarValue = Option.when(PlatformCompat.isJVM)(100.00)
```

That way, we expect an exemplar value only on the JVM.